part_2_dynamic.md

```diff
@@ -162,12 +162,12 @@ Option 2: Just bring it back, using `first` and `drop`
     </li>
   <% end %>
   <% @hobbies.drop(3).each do |hobby, i| %>
-    <li class="list-group-item">
+    <li class="list-group-item hide">
       <%= hobby %>
     </li>
   <% end %>
 </ul>
-{% endhighlight %})
```

Comment:
This is actually the closing parentheses of the whole optional block. I'll remove the opening one as well and merge this. After that we'll have to figure out which way we want to handle those optional, for quickly moving users - kind of blocks.